daprdocs/layouts/partials/page-meta-links.html

```diff
@@ -0,0 +1,32 @@
+{{ if .Path }}
+{{ $pathFormatted := replace .Path "\\" "/" }}
+{{ $gh_repo := ($.Param "github_repo") }}
+{{ $gh_subdir := ($.Param "github_subdir") }}
+{{ $gh_project_repo := ($.Param "github_project_repo") }}
+{{ $gh_branch := (default "master" ($.Param "github_branch")) }}
+{{ if $gh_repo }}
+<div class="td-page-meta ml-2 pb-1 pt-2 mb-0">
+{{ $gh_repo_path := printf "%s/content/%s" $gh_branch $pathFormatted }}
+{{ if and ($gh_subdir) (.Site.Language.Lang) }}
+{{ $gh_repo_path = printf "%s/%s/content/%s/%s" $gh_branch $gh_subdir ($.Site.Language.Lang) $pathFormatted }}
+{{ else if .Site.Language.Lang }}
+{{ $gh_repo_path = printf "%s/content/%s/%s" $gh_branch ($.Site.Language.Lang) $pathFormatted }}
+{{ else if $gh_subdir }}
+{{ $gh_repo_path = printf "%s/%s/content/%s" $gh_branch $gh_subdir $pathFormatted }}
+{{ end }}
+{{ $editURL := printf "%s/edit/%s" $gh_repo $gh_repo_path }}
+{{ $createURL := printf "%s/edit/%s" $gh_repo $gh_repo_path }}
+{{ $issuesURL := printf "%s/issues/new/choose" $gh_repo}}
+{{ $newPageStub := resources.Get "stubs/new-page-template.md" }}
+{{ $newPageQS := querify "value" $newPageStub.Content "filename" "change-me.md" | safeURL }}
+{{ $newPageURL := printf "%s/new/%s?%s"  $gh_repo $gh_repo_path $newPageQS }}
+
+<a href="{{ $editURL }}" target="_blank"><i class="fa fa-edit fa-fw"></i> {{ T "post_edit_this" }}</a>
+<a href="{{ $issuesURL }}" target="_blank"><i class="fab fa-github fa-fw"></i> {{ T "post_create_issue" }}</a>
+{{ if $gh_project_repo }}
+{{ $project_issueURL := printf "%s/issues/new/choose" $gh_project_repo }}
```

Comment:
Are we sure we want a "project issue" button? I'm wondering if that is the reason we saw some transferred issues from dapr/dapr - it might be easier and less confusing to just have "edit this page" and "open a doc issue"

Reply:
I kept it in because now that it points to the choose issue type page you can choose Feature request/discussion/etc. Thought it would be useful for users who read the documentation and want to go request a new feature

Reply:
Although it may not make sense for things like dashboard/components/etc. I'll disable it for now